frontend/pages/hosts/details/cards/HostSummary/_styles.scss

```diff
@@ -0,0 +1,13 @@
+.host-summary {
+  .info-flex {
+    &__header {
+      display: flex;
+      align-items: center;
+      .premium-icon-tip {
+        position: absolute;
```

Comment:
Is there a way we can use relative positioning in case there's a modification to the HostSummary down the line (like padding changes, adding a new summary header) that moves the Issues header?

Reply:
Yes. It was causing some layout issues which this approach avoided. We can either keep it like this, or I can use relative and deal with said issues case-by-case. I'll take this to mean you'd prefer it be relative, and make that change.